scorer/verifications/brightid.py

```diff
@@ -2,14 +2,14 @@
 from arango import ArangoClient
 
 
-def verify(graph):
-    print('BrightID')
+def verify(fname):
+    print('BRIGHTID')
     db = ArangoClient().db('_system')
     for u in db['users']:
         verifications = set([v['name'] for v in db['verifications'].find({'user': u['_key']})])
-        callJoined = 'CallJoined' in verifications
         seedConnected = 'SeedConnected' in verifications
-        if not (callJoined and seedConnected):
+        seedConnectedWithFriend = 'SeedConnectedWithFriend' in verifications
+        if not (seedConnected and seedConnectedWithFriend):
```

Comment:
Isn't it impossible to have `seedConnectedWithFriend` without also having `seedConnected`?

Reply:
yes, it should be impossible, but double checking does not hurt, yes?
there always may be bugs in those verifications so maybe it's better to keep that check there.